ts/Series/Heatmap/HeatmapSeries.ts

```diff
@@ -517,10 +515,8 @@ class HeatmapSeries extends ScatterSeries {
                         plotHeight
                     ).add(heatmap.group);
                 }
+                Series.prototype.drawPoints.call(heatmap);
```

Comment:
Back to drawing an SVG shape for each point? That will be slow for heatmaps with thousands of points. Have you considered using a k-d-tree for the tooltips? We already have the code for it in the boost module.